orsopy/fileio/data_source.py

```diff
@@ -61,15 +87,26 @@ class Sample(Header):
 
 @dataclass(repr=False)
 class InstrumentSettings(Header):
-    """Settings associated with the instrumentation."""
+    """
+    Settings associated with the instrumentation.
+
+    :param incident_angle: Angle (range) of incidence.
+    :param wavelength: Neutron/x-ray wavelenght (range).
+    :param polarization: Radiation polarization as one of
+        :code:`'unpolarized'`, :code:`'p'`, :code:`'m'`, :code:`'pp'`,
+        :code:`'pm'`, :code:`'mp'`, :code:`'mm'`, or a
+        :py:class:`orsopy.fileio.base.ValueVector`.
+    :param configuration: Description of the instreument configuration (full
+        polarized/liquid surface/etc).
+    """
     incident_angle: Union[Value, ValueRange]
     wavelength: Union[Value, ValueRange]
-    polarization: Optional[Union[Literal['unpolarized', 'p', 'm', 'mm', 'mp', 'pm', 'pp'],
```

Comment:
Why were the Literals removed? It basically means the creator of the file can supply any old string for polarization when we need them to be one of a small set. 
I would've thought we would want the typing for validation, so if we eventually want to generate the schema from this code, then the original typing needs to be kept.

Reply:
See above